target/xilinx/scripts/impl_ip.tcl

```diff
@@ -87,6 +87,43 @@ switch $proj {
                     CONFIG.CLKOUT4_PHASE_ERROR {89.971} \
                     ] [get_ips $proj]
             }
+            vcu118 {
```

Comment:
Here, if vcu118 is no different than vcu128, you can just add a case to the prior block (add `vcu118 -` above it, see tcl switch syntax).

If it *is* different, the question is why.

Same goes for all IP configurations below.